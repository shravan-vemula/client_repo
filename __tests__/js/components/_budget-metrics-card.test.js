import React from "react";
import { render, cleanup } from "@testing-library/react";
import BudgetMetricsCard from "js/_components/organisms/_budget-metrics-card";

afterEach(cleanup);

describe("Budget Metrics Card Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<BudgetMetricsCard />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should have all the icons", () => {
    const { getAllByRole } = render(
      <BudgetMetricsCard heading="Budget" value="80,000" />
    );
    expect(getAllByRole("img")).toBe;
  });
  it("should have heading and value", () => {
    const { getByText } = render(
      <BudgetMetricsCard heading="Budget" value="80,000" />
    );
    expect(getByText("Budget")).toBe;
    expect(getByText("80,000")).toBe;
  });
});
