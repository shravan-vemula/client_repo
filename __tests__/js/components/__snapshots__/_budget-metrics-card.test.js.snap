// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Budget Metrics Card Component renders correctly 1`] = `ShallowWrapper {}`;
